providers/prometheus/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus
```

Comment:
Since we want to release all modules once, we need v2 here.
```suggestion
module github.com/grpc-ecosystem/go-grpc-middleware/providers/openmetrics/v2
```

Reply:
Okay, changed!